functions/index.js

```diff
@@ -0,0 +1,80 @@
+const functions = require('firebase-functions');
+
+// The Firebase Admin SDK to access the Firebase Realtime Database.
+const admin = require('firebase-admin');
+admin.initializeApp();
+const db = admin.firestore();
+const userDataCollection = db.collection('userData');
+
+let average = (array) => array.reduce((a, b) => a + b) / array.length;
+
+function isOld (semester){
+  var currentTime = new Date();
+  var month = currentTime.getMonth() + 1;
+  var year = currentTime.getFullYear();
+  if(semester.year > year){
+    return false;
+  }
+  else if (semester.year < year){
+    return true;
+  }
+  else{
+    if(semester.month <= month){
```

Comment:
I don't believe semester's have a month parameter in Firebase. I think they have "type" of either spring, summer, winter, or fall. How does this work?